### DISCUSSION TITLE
IKEA E2201 Rodret dimmer doesn't expose brightness_move_down, and brightness_move_up

### DISCUSSION THREAD

**Discussion by Cloud-Tickler** [Koenkk/zigbee2mqtt#21224] [Devices]
Hello together,

I got three IKEA E2201 Rodret dimmer. Adding them worked successful. But I only  get "on" action and "off" action available. No brightness_move_down, and brightness_move_up. Has anybody had the same experience and knows a workaround? 

<img width="922" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/127838436/6dc080ad-f91a-4e57-a7f2-300939249211">

They have the newest firmware installed. I'm using a Sonoff ZBDongle-P.

Thanks in advance.

**Comment by jlj242** [comment]
They might not com visible until you actually press brightness down/up, i.e hold them pressed for a few seconds
